roles/openshift_metrics/tasks/install_hawkular.yaml

```diff
@@ -47,18 +47,34 @@
   slurp: src={{ openshift_metrics_certs_dir }}/ca.crt
   register: metrics_route_dest_ca_cert
 
-- name: generate the hawkular-metrics route
-  template:
-    src: route.j2
-    dest: "{{ mktemp.stdout }}/templates/hawkular-metrics-route.yaml"
-  vars:
-    name: hawkular-metrics
-    labels:
-      metrics-infra: hawkular-metrics
-    host: "{{ openshift_metrics_hawkular_metrics_hostname }}"
-    to:
-      kind: Service
+- block:
+  - local_action: shell mktemp
+    register: ca_tempfile
+    changed_when: False
+
+  - local_action: stat path={{openshift_metrics_hawkular_ca}}
+    register: hawkular_ca_file
+    changed_when: False
+
+  - local_action: shell cat {{openshift_metrics_hawkular_ca}} > {{ca_tempfile.stdout}}
+    when: hawkular_ca_file.stat.exists 
+
+  - name: generate the hawkular-metrics route
+    template:
+      src: route.j2
+      dest: "{{ mktemp.stdout }}/templates/hawkular-metrics-route.yaml"
+    vars:
       name: hawkular-metrics
-    tls:
-      termination: reencrypt
-      destination_ca_certificate: "{{ metrics_route_dest_ca_cert.content }}"
+      labels:
+        metrics-infra: hawkular-metrics
+      host: "{{ openshift_metrics_hawkular_hostname }}"
+      to:
+        kind: Service
+        name: hawkular-metrics
+      tls:
+        termination: reencrypt
+        ca_certificate: "{{ lookup('file',ca_tempfile.stdout) }}"
+        destination_ca_certificate: "{{ metrics_route_dest_ca_cert.content | b64decode }}"
+  always:
+  - local_action: file state=absent path={{ca_tempfile.stdout}}
+    changed_when: False
```

Comment:
Of course, I forgot the file may not exist.  I'd suggest this, then, to avoid
creating temporary files (with similar tasks for the key and cert):

```yaml
- set_fact: hawkular_ca={{ lookup('file', openshift_metrics_hawkular_ca }}
  when: openshift_metrics_hawkular_ca|exists
# …
- name: generate the hawkular-metrics route
  template:
# …
      ca_certificate: "{{ hawkular_ca|default('') }}"
```